test/s3select_test.cpp

```diff
@@ -238,8 +238,8 @@ TEST(TestS3SElect, intnull_compare_operator)
   ASSERT_EQ( d >= c10, false );
   ASSERT_EQ( d < a10, false );
   ASSERT_EQ( d <= b11, false );
-  ASSERT_EQ( d != a10, true );
-  ASSERT_EQ( d != e, true );
+  ASSERT_EQ( d != a10, false );
+  ASSERT_EQ( d != e, false );
```

Comment:
ok,
comparison with NULL is UNKNOWN which is null 
but this test , is testing the /operator !=/
should test in s3-select-query